clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -679,4 +573,130 @@ public void patch(
       }
     }
   }
+
+  /**
+   * Handles validating whether kubernetes kinds should be cached by the current account, and
+   * whether metrics are enabled for the account.
+   */
+  private class KindValidator {
+    private final Supplier<String> checkNamespace = Suppliers.memoize(this::computeCheckNamespace);
```

Comment:
nit: a lot of people (including me, but certainly not everyone) leave out `private` visibility declarations inside private classes, since they're arguably redundant noise. But up to you.

Java Practices seems to take no real position: go/java-practices/redundancy#visibility-specifiers-in-private-classes

Reply:
Just chatted about this; in this case I added `private` to anything not intended to be called from outside the inner class, and left the two methods intended to be called from the outer class as package private.